app/javascript/packs/ssn-field.ts

```diff
@@ -34,7 +33,7 @@ function formatSSNFieldAndLimitLength() {
       sync();
       toggle.addEventListener('change', sync);
 
-      function limitLength() {
+      function limitLength(this: HTMLInputElement) {
```

Comment:
Future refactoring opportunity: `this` is probably more trouble than it's worth, and we could either pass `input` as a parameter or reference `input` directly from the parent scope.